src/lib/import_export/binary/binary_parser.cpp

```diff
@@ -109,6 +109,10 @@ void BinaryParser::_import_chunk(std::ifstream& file, std::shared_ptr<Table>& ta
 
   const auto mvcc_data = std::make_shared<MvccData>(row_count, CommitID{0});
   table->append_chunk(output_segments, mvcc_data);
+
+  if (finalize_chunk) {
+    table->get_chunk(ChunkID{table->chunk_count() - 1})->finalize();
```

Comment:
Can we use the table's `last_chunk()` method here?

Reply:
Ah ... wasn't aware of it. Will change.